andela/travel_tool_front
# #161345105 Delete travel team member

#### What does this PR do?
This PR deletes travel team members

#### Description of Task to be completed?
- Create delete role modal
- Create actions, reducers, and sagas to `delete travel team members`
- Write the test for implementation

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the`DB` and on the `UserRoles` table, set yourself as a `Travel Admin`
- Go to the browser, refresh the page, Navigate to the `Settings` tab and  click on `Travel Team`
- Add yourself as a ` travel team member`
- Click the `delete` button on your role
- A `confirmation modal` will pop up
- `confirm delete`

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161345105](https://www.pivotaltracker.com/story/show/161345105)

#### Screenshots (if appropriate)
<img width="1433" alt="screen shot 2018-10-25 at 2 23 42 pm" src="https://user-images.githubusercontent.com/27569442/47503317-b4b52d80-d861-11e8-9855-77c486079bb6.png">
<img width="1440" alt="screen shot 2018-10-25 at 2 24 20 pm" src="https://user-images.githubusercontent.com/27569442/47503347-c3034980-d861-11e8-9e41-f127ba1c62dd.png">


#### Questions:
N/A
